test/linear_solver_routines/block_solver_helpers/proc_gentlemans_house.jl

```diff
@@ -0,0 +1,49 @@
+# This file is part of RLinearAlgebra.jl
+
+module ProceduralTestGent
+
+using Test, RLinearAlgebra, LinearAlgebra, Random
+
+@testset "Gent Projection -- Procedural" begin
+    A = rand(10,5)
+    x = rand(5)
+    b = A * x
+    G = RLinearAlgebra.GentData(A, 3)
+    # Check sizes of block matrix B 
+    m, n = size(G.B)
```

Comment:
Needs a comment explaining why these numbers are what they are. There should a column appended in QR, and then there needs to be a bugger for the additional equations (e.g., 3)

Reply:
I have added this comment